main.py

```diff
@@ -74,6 +84,79 @@ def player_set_volume():
     return jsonify({'message': 'No volume parameter'}), 400
 
 
+@app.route('/v1/equalizer_info', methods=['GET'])
```

Comment:
Let's put all of these endpoints under `/v1/player/equalizer/`. So something along the lines of `/v1/player/equalizer/info`, `/v1/player/equalizer/enable`, etc.